src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,214 @@
+    case 'privacy':
+      return ['{446900e4-71c2-419f-a6a7-df9c091e268b}'];
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return null;
+  }
+};
+
+export const getContent = (guidesSlug: string, i18n: I18nType): Object => {
+  switch (guidesSlug) {
+    case 'privacy':
+      return {
+        title: i18n.gettext('Stay Safe Online'),
+        introText: i18n.gettext(
+          'The web is a wonderful but wild place. Your personal data can be used without your consent, your activities spied upon, and your passwords stolen. Fortunately, extensions can help fortify your online privacy and security.',
+        ),
+        icon: 'stop-hand',
+        sections: [
+          // Bitwarden free password manager
+          {
+            header: i18n.gettext('Create and manage strong passwords'),
+            description: i18n.gettext(
+              'Password managers can help you create secure passwords, store your passwords (safely) in one place, and give you easy access to your login credentials wherever you are.',
+            ),
+            addonCustomText: i18n.gettext(
+              'Fully encrypted password protection. Store your data securely and access logins across devices.',
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/password-managers/',
+          },
+        ],
+      };
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return {};
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { guidesSlug, guids } = this.props;
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: guidesSlug,
+      }),
+    );
+  }
+
+  getGuidesSectionsHtml = (
+    sections: Array<Object>,
+  ): React.ChildrenArray<React.Node> => {
+    const { addons, clientApp, i18n, lang } = this.props;
+
+    return addons.length
+      ? addons.map((addon, key) => {
+          // TODO: look into having these links use the Router (vs 'a' tag).
+          // See https://github.com/mozilla/addons-frontend/issues/6787.
+          const exploreMore = i18n.sprintf(sections[key].exploreMore, {
+            linkStart: `<a href="/${lang}/${clientApp}${
+              sections[key].exploreUrl
+            }">`,
+            linkEnd: '</a>',
+          });
+
+          return (
+            <div className="Guides-section" key={addon && addon.url}>
+              <h2 className="Guides-section-title">{sections[key].header}</h2>
```

Comment:
Would it make sense to use a map for `sections`? We could use add-on's `guid`s as keys.

Reply:
yeh I like that..I can map through sections instead.

Reply:
Altho this might come up again when I need to reference the affiliated addon...

Reply:
maybe we can chat about this on weds?

Reply:
What are affiliated add-ons?

Reply:
each section item corresponds with its own addon

Reply:
ah yep, so each section can also be mapped by add-ons `guid`, isnt it?

Reply:
yeh as I mentioned below, I will add a guid to each section. I think this way is definitely clearer :)